website_apps_store/controllers/main.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2017-2018 BizzAppDev
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import base64
+
+from odoo import http
+from odoo.http import request
+from odoo.addons.http_routing.models.ir_http import slug
+from odoo.addons.website.controllers.main import QueryURL
+from odoo.addons.website_sale.controllers.main import WebsiteSale, TableCompute
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+PPG = 20  # Products Per Page
+PPR = 4   # Products Per Row
+
+
+class WebsiteSaleCustom(WebsiteSale):
+
+    def _get_search_domain(self, search, category, attrib_values):
+        domain = request.website.sale_product_domain()
+        if search:
+            for srch in search.split(" "):
+                domain += [
+                    '|', '|', '|', '|', '|', '|', '|', '|',
+                    ('name', 'ilike', srch),
+                    ('technical_name', 'ilike', srch),
+                    ('description', 'ilike', srch),
+                    ('description_sale', 'ilike', srch),
+                    ('product_variant_ids.default_code', 'ilike', srch),
+                    ('product_variant_ids.attribute_value_ids.name',
+                     'ilike', srch),
+                    ('product_variant_ids.app_description_rst_html',
+                     'ilike', srch),
+                    ('product_variant_ids.app_author_ids.name', 'ilike', srch),
+                    ('product_variant_ids.app_summary', 'ilike', srch)]
+        if category:
+            domain += [('public_categ_ids', 'child_of', int(category))]
+
+        if attrib_values:
+            attrib = None
+            ids = []
+            for value in attrib_values:
+                if not attrib:
+                    attrib = value[0]
+                    ids.append(value[1])
+                elif value[0] == attrib:
+                    ids.append(value[1])
+                else:
```

Comment:
Why need this? This do the same that "if not attrib" but split the domain in more 'criterias' no?

All conditions do the same... add to ids the "value[1]".

Reply:
As I said this PR scope is migration